data/brands/shop/cosmetics.json

```diff
@@ -577,6 +578,34 @@
         "shop": "cosmetics"
       }
     },
+    {
+      "displayName": "Рив Гош",
+      
+      "locationSet": {"include": ["ru"]},
+      "tags": {
+        "brand": "Рив Гош",
+        "brand:fr": "Rive Gauche",
+        "brand:wikidata": "Q4394330",
+        "brand:wikipedia": "ru:Рив_Гош",
+        "name": "Рив Гош",
+        "name:ru": "Рив Гош",
+        "name:fr": "Rive Gauche"
+        "shop": "cosmetics"
+      }
+    },
+    {
+      "displayName": "Улыбка радуги",
+      
+      "locationSet": {"include": ["ru"]},
+      "tags": {
+        "brand": "Улыбка радуги",
+        "brand:wikidata": "Q109734104",
+        "name": "Улыбка радуги",
+        "name:ru": "Улыбка радуги",
+        "shop": "cosmetics"
+      }
+    },
+    
```

Comment:
Extra line
```suggestion
```

Reply:
sorry this extra line is made by me……i will fix It on PC by using nodejs script run one time 😭